adding index.yaml and kuberhealthy-2.1.0.tgz as req'd for helm repo

chadbitzer opened the pull request:
adding index.yaml and kuberhealthy-2.0.0.tgz at root of Comcast/kuberhealthy

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Comcast/kuberhealthy?pullRequest=379) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Comcast/kuberhealthy?pullRequest=379) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Comcast/kuberhealthy?pullRequest=379) it.</sub>

chadbitzer commented:
created dirs helm-repos & helm-repos/kuberhealthy at root of kuberhealthy repo, added index.yaml with modified relative url for .tgz, index.md to identify repo in browser, and moved .tgz package into helm-repos/kuberhealthy.

chadbitzer commented:
updated Charts.yaml with correct Kuberhealthy 2.1.0 versioning, repackaged charts, and regenerated index.yaml.

chadbitzer commented:
standby on code review, "helm install" from a mirrored repo is failing to download.
